### PR TITLE
[CAS-1220] Numerous improvements

### PR DESCRIPTION
**Pull request by bychkovdmitry**

### CODE REVIEW

File: virtualevent-demo/src/main/res/values/themes.xml [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
         <item name="colorPrimary">@color/white</item>
         <item name="colorPrimaryVariant">@color/white</item>
         <item name="colorOnPrimary">@color/white</item>
         <item name="colorSecondary">@color/gray</item>
         <item name="colorSecondaryVariant">@color/gray</item>
         <item name="colorOnSecondary">@color/gray</item>
-        <item name="android:statusBarColor" tools:targetApi="l">?attr/colorSecondary</item>
+        <item name="android:navigationBarColor">@color/app_canvas</item>
+        <item name="android:statusBarColor">@color/app_canvas</item>
+        <item name="android:windowLightStatusBar">true</item>
```
Comment (bychkovdmitry) [inline virtualevent-demo/src/main/res/values/themes.xml:12]: Changing status bar and nav bar colors to white

File: virtualevent-demo/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -1,22 +1,56 @@
...
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:textColor">@color/text_mid_emphasis</item>
+        <item name="android:layout_marginStart">@dimen/margin_small</item>
+    </style>
+
+    <style name="OverviewSectionTitle">
+        <item name="android:textColor">@color/text_high_emphasis</item>
+        <item name="android:textAppearance">@style/Title3Bold</item>
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">wrap_content</item>
```
Comment (bychkovdmitry) [inline virtualevent-demo/src/main/res/values/styles.xml:33]: I know that having layout attrs in a style is a bad practice. Doing this for simplicity.

File: virtualevent-demo/src/main/res/drawable/bg_badge_stats.xml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android">
-    <solid android:color="#E5E5E5" />
+    <solid android:color="@color/card_background" />
```
Comment (bychkovdmitry) [inline virtualevent-demo/src/main/res/drawable/bg_badge_stats.xml:3]: All the colors and their names are taken from the color palette in Figma

File: virtualevent-demo/src/main/java/io/getstream/chat/virtualevent/AppConfig.kt [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+            "name" to "Samuel",
+            "image" to "https://firebasestorage.googleapis.com/v0/b/stream-chat-internal.appspot.com/o/users%2FJc.png?alt=media",
+        ),
+    )
+    const val USER_TOKEN =
+        "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoic2FtdWVsX3VyYmFub3dpY3oifQ.Zsn_9f5LbvyV9F5jQ-_h7YvQNnNsnUQ6_IMRMemA334"
+
+    const val LIVESTREAM_CHANNEL_1 = "livestream:esg_data_f22160f7-01fd-423f-b622-fe7060ec10d8"
+    const val LIVESTREAM_CHANNEL_2 = "livestream:data-strategy_cfe253a5-785b-4c77-a5a0-ec63693d4e58"
+}
```
Comment (bychkovdmitry) [inline virtualevent-demo/src/main/java/io/getstream/chat/virtualevent/AppConfig.kt:20]: Kind of a mess here ATM

File: virtualevent-demo/build.gradle [hunk 5]
```diff
@@ -1,7 +1,6 @@
 plugins {
     id 'com.android.application'
     id 'kotlin-android'
-    id 'androidx.navigation.safeargs.kotlin'
 }
```
Comment (bychkovdmitry) [inline virtualevent-demo/build.gradle:4]: No longer needed as we only use jetpack navigation for switching between tabs. No args are passed to them.

File: virtualevent-demo/src/main/java/io/getstream/chat/virtualevent/feature/dm/list/DirectChatsFragment.kt [hunk 6]
```diff
@@ -42,12 +49,13 @@ class DMFragment : Fragment() {
...
-            startActivity(DmActivity.createIntent(requireContext(), channel.cid))
+            startActivity(DirectChatActivity.createIntent(requireContext(), channel.cid))
         }
         binding.startChatButton.setOnClickListener {
-            startActivity(SelectUserActivity.createIntent(requireContext()))
+            startActivity(StartDirectChatActivity.createIntent(requireContext()))
         }
-        binding.channelListView.setViewHolderFactory(DmChannelListViewHolderFactory())
+        // TODO: implement custom direct chat ViewHolder factory
+        // binding.channelListView.setViewHolderFactory(DirectChatListItemVhFactory())
```
Comment (bychkovdmitry) [inline virtualevent-demo/src/main/java/io/getstream/chat/virtualevent/feature/dm/list/DirectChatsFragment.kt:58]: Will do it later

File: virtualevent-demo/src/main/java/io/getstream/chat/virtualevent/feature/dm/start/ParticipantListAdapter.kt [hunk 7]
```diff
@@ -54,4 +53,33 @@ class UserListAdapter(
...
+            "Nolan Inc",
+            "Keebler LLC",
+            "Huels - Daniel"
+        )
+
+        /**
+         * Returns a random company name.
+         */
+        fun randomCompany(): String = COMPANY_LIST.random()
+    }
```
Comment (bychkovdmitry) [inline virtualevent-demo/src/main/java/io/getstream/chat/virtualevent/feature/dm/start/ParticipantListAdapter.kt:84]: Probably need to remove this

**Review by samiuelson — APPROVED** [review]
Great cleanup!

**Pull request merged**